src/Mod/Path/PathScripts/PathFeatureExtensionsGui.py

```diff
@@ -299,16 +301,33 @@ def _initializeExtensions(self, obj):
             self.form.extensionEdit.setDisabled(True)
         self.setExtensions(self.extensions)
 
-    def updateQuantitySpinBoxes(self, index=None):
-        prevValue = self.form.defaultLength.text()
-        self.defaultLength.updateSpinBox()
-        postValue = self.form.defaultLength.text()
+    def _isDefaultLengthExpression(self):
```

Comment:
nitpick: the name here is a bit confusing because it's not clear what _is_ the expression. I think it would read and could be processed easier if you would name it `_haveDefaultLengthExpression(self):`  or `_isDefaultLengthExpressionAssigned` - something along those lines.

Reply:
Made some improvements.  I moved the `valueChanged` signal translation into the QuantitySpinBox class in the PathGui module, as that is where it belongs.  This will fix each instance where the QSB class is deployed.  It also simplified the overall fix for this issue related to extensions updating.